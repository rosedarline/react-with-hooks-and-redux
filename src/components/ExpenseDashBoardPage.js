import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";
import ExpensesSummary from "./ExpensesSummary";


function ExpenseDashBoardPage () {
    return (
        <div>
            <ExpensesSummary />
            <ExpenseListFilters />
            <ExpenseList />
        </div>
    );
};


export default ExpenseDashBoardPage;