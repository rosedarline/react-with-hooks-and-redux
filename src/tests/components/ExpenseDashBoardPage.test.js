import React from "react";
import { shallow } from "enzyme";
import ExpenseDashBoardPage from "../../components/ExpenseDashBoardPage";

test("Should render ExpenseDashBoard correctly", () => {
    const wrapper = shallow(<ExpenseDashBoardPage />);
    expect(wrapper).toMatchSnapshot();
});