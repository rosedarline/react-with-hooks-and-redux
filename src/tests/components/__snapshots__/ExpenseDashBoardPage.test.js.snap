// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseDashBoard correctly 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
